src/containers/AudioUploader/components/AudioContent.jsx

```diff
@@ -9,36 +9,47 @@
 import React, { Fragment } from 'react';
 import PropTypes from 'prop-types';
 import { injectT } from '@ndla/i18n';
-import { InputFileField, TextField } from '../../../components/Fields';
-import { CommonFieldPropsShape } from '../../../shapes';
 import AudioPlayer from './AudioPlayer';
+import FormikField from '../../../components/FormikField';
 
-const AudioContent = ({ t, commonFieldProps, model, audioInfo }) => (
+const AudioContent = ({ t, values, audioInfo, setFieldValue }) => (
   <Fragment>
-    <TextField
+    <FormikField
       label={t('form.title.label')}
       name="title"
       title
       noBorder
       placeholder={t('form.title.label')}
-      {...commonFieldProps}
     />
-    {!model.id && (
-      <InputFileField
-        label={t('form.audio.file')}
+    {!values.id && (
+      <FormikField
+        id="file"
+        noBorder
         name="audioFile"
-        {...commonFieldProps}
-      />
+        label={t('form.audio.file')}>
+        {({ field }) => (
+          <input
+            type="file"
```

Comment:
Tror du kan bare sende inn type og onChange direkte til <FormikField>, for da blir det sendt inn til Formik sin egen Input komponent

Reply:
direkte til hva? Tror du må forklare denne bedre